envs/secrets.env.template

```diff
@@ -6,3 +6,5 @@ API_MAILJET_SECRET=set_it
 
 API_EMPLOI_STORE_KEY=set_it
 API_EMPLOI_STORE_SECRET=set_it
+
+PDFSHIFT_API_KEY=set_it
```

Comment:
*No newline at end of line.* Tu as tendance à les oublier partout.

Reply:
C'est vraiment bizarre car pylint ne me le dit pas. J'ai pourtant un hook git qui lance black et pylint à chaque commit. Bref, je vais l'ajouter.

Reply:
Intéressant, tu peux peut-être partager ton hook existant dans https://trello.com/c/lJx6iZjr/353-int%C3%A9grer-isort-flake8-et-black-dans-un-pre-commit-hook ?